legal-api/report-templates/template-parts/transition/businessDetails.html

```diff
@@ -0,0 +1,31 @@
+<div class="business-details-container">
```

Comment:
This section need not be a new file. It can be any of the existing business details I think the only difference is Transition date which you can definitely add based on the type of filing